includes/components/class-admin.php

```diff
@@ -946,6 +949,312 @@ protected function get_extensions_tab() {
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s version', 'hivepress' ), 'WordPress' ),
+							'label_en' => 'WordPress version',
+							'value'    => get_bloginfo( 'version' ),
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s multisite', 'hivepress' ), 'WordPress' ),
+							'label_en' => 'WordPress multisite',
+							'value'    => is_multisite() ? '+' : '-',
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s memory limit', 'hivepress' ), 'WordPress' ),
+							'label_en' => 'WordPress memory limit',
+							'value'    => WP_MEMORY_LIMIT,
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s debug mode', 'hivepress' ), 'WordPress' ),
+							'label_en' => 'WordPress debug mode',
+							'value'    => ( defined( 'WP_DEBUG' ) && WP_DEBUG ) ? '+' : '-',
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s cron', 'hivepress' ), 'WordPress' ),
+							'label_en' => 'WordPress cron',
+							'value'    => ! ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) ? '+' : '-',
+						],
+						[
+							'label'    => esc_html__( 'Language' ),
+							'label_en' => 'Language',
+							'value'    => get_locale(),
+						],
+						[
+							'label'    => esc_html__( 'External object cache', 'hivepress' ),
+							'label_en' => 'External object cache',
+							'value'    => wp_using_ext_object_cache() ? '+' : '-',
+						],
+					],
+				],
+				[
+					'label'    => esc_html__( 'Server environment', 'hivepress' ),
+					'label_en' => 'Server environment',
+					'values'   => [
+						[
+							'label'    => esc_html__( 'Server info', 'hivepress' ),
+							'label_en' => 'Server info',
+							'value'    => isset( $_SERVER['SERVER_SOFTWARE'] ) ? sanitize_text_field( wp_unslash( $_SERVER['SERVER_SOFTWARE'] ) ) : '-',
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s version', 'hivepress' ), 'PHP' ),
+							'label_en' => 'PHP version',
+							'value'    => phpversion(),
+						],
+						[
+							'label' => 'PHP post max size',
+							'value' => ini_get( 'post_max_size' ),
+						],
+						[
+							'label' => 'PHP time limit',
+							'value' => (int) ini_get( 'max_execution_time' ),
+						],
+						[
+							'label' => 'PHP max input vars',
+							'value' => (int) ini_get( 'max_input_vars' ),
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s version', 'hivepress' ), 'cURL' ),
+							'label_en' => 'cURL version',
+							'value'    => $curl_version,
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s installed', 'hivepress' ), 'SUHOSIN' ),
+							'label_en' => 'SUHOSIN installed',
+							'value'    => extension_loaded( 'suhosin' ) ? '+' : '-',
+						],
+						[
+							'label'    => sprintf( esc_html__( '%s version', 'hivepress' ), 'MySQL' ),
+							'label_en' => 'MySQL version',
+							'value'    => $wpdb->db_server_info(),
+						],
+						[
+							'label' => 'Max upload size',
```

Comment:
Also unstranslatable. I'm not sure why most of the comments were ignored during the first check, and on the second one the same issues from the initial comments remain